src/main/webapp/resources/js/components/samples/components/SampleFiles.tsx

```diff
@@ -299,55 +372,113 @@ export function SampleFiles() {
               props={{ className: "t-upload-sample-files" }}
             />
           </div>
-          <div>
-            <SampleFileConcatenate>
-              <Button
-                className="t-concatenate-btn"
-                disabled={concatenateSelected?.length < 2}
-              >
-                {i18n("SampleFiles.concatenate")}
-              </Button>
-            </SampleFileConcatenate>
-          </div>
+          {concatenateFunctionality()}
         </Col>
-      ) : null}
-      {sequenceFiles.length || assemblyFiles.length || fast5Files.length ? (
+      );
+    }
+  };
+
+  /*
+  Function to determine if the concatenate functionality should be visible to the user or not
+   */
+  const concatenateFunctionality = () => {
+    if (
+      (files.singles !== undefined && files?.singles?.length >= 2) ||
+      (files.paired !== undefined && files?.paired?.length >= 2)
+    ) {
+      return (
+        <div>
```

Comment:
Why the surrounding div?

Reply:
I seem to have some kind of weird obsession with surrounding stuff with divs

Reply:
Removed in 85b90900deaddee54dbae6a8198ae5b1b77a577c